camel-petals/src/main/java/org/ow2/petals/camel/helpers/PetalsRouteBuilder.java

```diff
@@ -27,6 +29,66 @@
 
 public abstract class PetalsRouteBuilder extends RouteBuilder {
 
+    private final Properties placeholders = new Properties();
+
+    /**
+     * Camel route definition call-back called just after the service-unit deployment and before its initialization.
+     */
+    public void deploy() {
```

Comment:
add a `throws Throwable` to all the methods…

Reply:
Added `throws Exception`